1/assets/1.1-finished-code.rs

```diff
@@ -28,9 +26,11 @@ mod incrementer {
     mod tests {
         use super::*;
 
-        #[test]
+        use ink_lang as ink;
+
+        #[ink::test]
```

Comment:
```suggestion
        #[test]
```

Reply:
The special `#[ink::test]` proc macro is only required if the test interacts with the environment, e.g. wants to receive caller, transferred balance etc. This does not count for storage reads/writes though.